src/components/Chip/chip.module.scss

```diff
@@ -1,12 +1,14 @@
 .chip {
   display: flex;
   align-content: center;
-  border: 1px solid var(--cl-text-light);
+  border: 1px solid whitesmoke;
```

Comment:
Please don't use the predefined CSS color names, but the exact color in the design, you can find it in Figma. Preferably formatted as hex code (any other format would also be fine, as long as it is the exact color in the design, we just use hex codes in the rest of the styles).